.github/CHANGELOG.md

```diff
@@ -8,12 +8,21 @@
 
 ### Documentation
 
+* The documentation theme has been updated, and the documentation structure
+  reorganized.
+  [(#60)](https://github.com/XanaduAI/pennylane-pq/pull/60)
+
 ### Bug fixes
 
+* Updated the plugin to use the latest IBMQBackend from ProjectQ.
+  [(#62)](https://github.com/XanaduAI/pennylane-pq/pull/61)
+
 ### Contributors
 
 This release contains contributions from (in alphabetical order):
 
+Maria Schuld
+
```

Comment:
Need to add yourself @josh146 :)

Reply:
😆 Oddly enough, the tests now pass for me locally, but not on travis...

Reply:
Is it possible that Travis' token expired?